portfolio/src/main/webapp/index.html

```diff
@@ -143,24 +141,35 @@ <h2 id = "gallery" > Gallery </h2>
                 <img class = "slide" src="/images/group3.jpg">
             </div>
 
+            <div class = "mySlides">
```

Comment:
Nit: I think CSS class names are usually spelled in singular and lowercase + hyphen as opposed to camel case. 

This one could be a little more descriptive, like `slide-container` or maybe make this one `slide` and the inner one `slide-image`? Otherwise it's hard to tell in the CSS which one is `mySlides` and which is `slide`.